### DISCUSSION TITLE
get the selected item from dropdown

### DISCUSSION THREAD

**Discussion by hemna** [fomantic/Fomantic-UI#2579] [General] [answered]
The documentation shows how to create static dropdowns, but doesn't show how to get the selected item from js, and also doesn't show how to update the items in the dropdown list.  It would be helpful if the documentation showed not only static examples (as it does today), but also how to modify elements with JS and get values out of user selectable items from js as well.

Here is my static dropdown

```
<div class="ui fluid labeled icon dropdown button" id="test_dropdown">
                  <i class="comment icon"></i>
                  <span class="text">Message To/From....</span>
                  <div class="menu" id="message_menu">
                    <div class="header">
                      <i class="comment icon"></i>
                      Select Conversation
                    </div>
                    <div class="item">
                      KM6LYW
                    </div>
                    <div class="item">
                      WB4BOR-11
                    </div>
                    <div class="item">
                      REPEAT
                    </div>
                  </div>
                </div>
```

I want to be able to add more items on the fly, and I also need to get the current selection and when the user changes the selection from js.   How do I do this with jquery?

**Comment by jamessampford** [comment, marked as answer]
> I want to be able to add more items on the fly

Think you’d need to re-init the dropdown with the values as per https://fomantic-ui.com/modules/dropdown.html#initializing-with-javascript-only or `$(‘.ui.dropdown’).dropdown(‘change values’, values);` - i don’t think there’s anything to append on an item

>I also need to get the current selection and when the user changes the selection from js. How do I do this with jquery?

Can use the `onChange` callback otherwise can use `$(‘.ui.dropdown’).dropdown(‘get value’);`
